__tests__/port.test.js

```diff
@@ -1,13 +1,35 @@
-const Port = require('../src/port.js')
+const Port = require('../src/port')
+describe ('Port', () => {
+    let manchester;
+    let ship;
+    let david;
+    let alpha;
 
+beforeEach(() => {
+    manchester = new Port ('Manchester');
+    ship = jest.fn();
+    david = jest.fn();
+    alpha = jest.fn();
+})
+describe('constructor', () => {
```

Comment:
should be an empty line between the `beforeEach...` block and the `describe...` block